.github/workflows/build.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     - name: Check out repository
       uses: actions/checkout@v2
 
-    - name: Set up .NET Core
+    - name: Set up .NET
       uses: actions/setup-dotnet@v1
```

Comment:
setup-dotnet appears to have released a v4.  should we be using that?  same with the checkout a couple lines above?